src/components/organisms/TemplateExplorerPane/TemplateInformation.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+
+import {Button, Popconfirm} from 'antd';
+
+import {DeleteOutlined, ExclamationOutlined, FormOutlined} from '@ant-design/icons';
+
+import styled from 'styled-components';
+
+import {AnyTemplate} from '@models/template';
+
+import {useAppDispatch, useAppSelector} from '@redux/hooks';
+import {
+  deletePlugin,
+  deleteStandalonTemplate,
+  deleteTemplatePack,
+  isPluginTemplate,
+  isStandaloneTemplate,
+  isTemplatePackTemplate,
+} from '@redux/services/templates';
+
+import Colors from '@styles/Colors';
+
+const Container = styled.div`
+  display: grid;
+  grid-template-columns: max-content 1fr;
+  position: relative;
+  margin-bottom: 16px;
+`;
+
+const IconContainer = styled.span`
+  height: 50px;
+  width: 50px;
+`;
+
+const InfoContainer = styled.span`
+  display: flex;
+  flex-direction: column;
+`;
+
+const Name = styled.span`
+  font-weight: 600;
+`;
+
+const Description = styled.span`
+  font-weight: 300;
+`;
+
+const Footer = styled.span`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const Author = styled.span`
+  color: ${Colors.grey500};
+`;
+
+const Version = styled.span`
+  font-style: italic;
+`;
+
+const StyledDeleteOutlined = styled(DeleteOutlined)`
+  position: absolute;
+  top: 5px;
+  right: 0px;
+  color: ${Colors.red7};
+  cursor: pointer;
+`;
+
+const StyledFormOutlined = styled(FormOutlined)`
```

Comment:
Same as above, have all styled components in a separate `styled` file?

Reply:
Yes, will do